azure-pipelines-templates/device-test.yml

```
@@ -0,0 +1,154 @@
+        - task: DownloadPipelineArtifact@2
+          displayName: 'Download Pipeline Artifact'
+          inputs:
+            path: '$(System.DefaultWorkingDirectory)'
+
+        - bash: |
+            # ensure nanoff is the last version
+            dotnet tool install nanoff --tool-path /azp/tools
+          displayName: 'Install nanoff'
+
+        - ${{ each tests in parameters.unitTestRunsettings }}:
+          - bash: |
```

yamllint

[warning] wrong indentation: expected 12 but found 10

(indentation)